prov/rxm/src/rxm.h

```diff
@@ -506,6 +513,29 @@ void rxm_ep_progress_multi(struct util_ep *util_ep);
 int rxm_ep_prepost_buf(struct rxm_ep *rxm_ep, struct fid_ep *msg_ep);
 int rxm_send_queue_init(struct rxm_ep *rxm_ep, struct rxm_send_queue **send_queue, size_t size);
 void rxm_send_queue_close(struct rxm_send_queue *send_queue);
+void rxm_ep_progress_deferred_queues(struct rxm_ep *rxm_ep);
+
+static inline void
+rxm_ep_enqueue_deferred_tx_queue(struct rxm_tx_entry *tx_entry)
+{
+	if (dlist_empty(&tx_entry->deferred_tx_entry)) {
+		dlist_insert_tail(&tx_entry->conn->deferred_tx_queue,
+				  &tx_entry->deferred_tx_entry);
+		if (dlist_empty(&tx_entry->conn->deferred_conn_entry))
+			dlist_insert_tail(&tx_entry->ep->deferred_conn_queue,
+					  &tx_entry->conn->deferred_conn_entry);
+	}
+}
+
+static inline void
+rxm_ep_dequeue_deferred_tx_queue(struct rxm_tx_entry *tx_entry)
+{
+	if (!dlist_empty(&tx_entry->deferred_tx_entry)) {
+		dlist_remove_init(&tx_entry->deferred_tx_entry);
+		if (dlist_empty(&tx_entry->conn->deferred_tx_queue))
+			dlist_remove_init(&tx_entry->conn->deferred_conn_entry);
+	}
```

Comment:
Similar to above comment, tx_entry should be on the queue:
```
dlist_remove(tx_entry->entry);
if (dlist_empty(conn->deferred_queue))
    dlist_remove(conn->deferred_entry);
```

Reply:
Done. Thanks!